### ISSUE TITLE
Page numbering

### ISSUE THREAD

**Issue by Tyrn** [aarneng/Outline-Summaryst#1]
1. The page number in a TOC entry is correct; it doesn't show on pages.
2. After adding
```
#set page(numbering: "1")
```
before the first `make-heading()` the page numbering starts with page 2.  If `#set page(numbering: "1")` is placed before or after `#show outline`, the numbering starts on the first TOC page, then resets to page 2 again.

**Comment by aarneng** [comment]
Yeah, numbering is a big janky. You can try using branch `unstable-dev`  while I try to find a more reliable solution
